runtime/src/bank.rs

```diff
@@ -2265,6 +2283,30 @@ impl Bank {
         }
     }
 
+    fn burn_vat_from_staked_accounts(&mut self, epoch_stakes: &VersionedEpochStakes) {
+        let mut total_vat = 0;
+        let accounts_to_store = epoch_stakes
+            .stakes()
+            .staked_nodes()
+            .iter()
+            .filter_map(|(vote_pubkey, stake)| {
+                if *stake == 0u64 {
+                    None
+                } else {
+                    // burn VAT from each staked vote account
+                    let vat = ALPENGLOW_VAT_TO_BURN_PER_EPOCH;
+                    let mut account = self.get_account(vote_pubkey).unwrap();
+                    total_vat += vat;
+                    account.set_lamports(account.lamports().saturating_sub(vat));
```

Comment:
nit: IIUC at this point all of these accounts should have `>= ALPENGLOW_VAT_TO_BURN_PER_EPOCH` in the vote account:
i would prefer to `checked_sub(vat).unwrap()`